docs/development/compare_atomic_files.ipynb

```diff
@@ -6,145 +6,8878 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    pat = "/home/atharva/workspace/code/tardis-main/carsus/kurucz_cd23_chianti_He_cmfgen_H_Si_I-II.h5"
</pre><p>should not be local names and it should be path</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/carsus/pull/315/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Done in 829e752fe4d77b6c0c875896a12bfe806c9daaa9. Please see the diff [here](https://app.reviewnb.com/tardis-sn/carsus/commit/829e752fe4d77b6c0c875896a12bfe806c9daaa9/).